front/lib/models/assistant/actions/retrieval.ts

```diff
@@ -353,13 +352,14 @@ RetrievalDocument.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    // @ts-expect-error deprecated
```

Comment:
You this is working right? Because you removed the property RetrievalDocument.timetamp line 315.

Reply:
yes that's on purpose. next PR I actually remove the column, but first I remove from the typesystem, remove usage and deploy 👍